tests/core/test_core.py

```diff
@@ -124,18 +125,22 @@ def test_narrow_to_stream(
         mocker: MockerFixture,
         controller: Controller,
         index_stream: Index,
+        general_stream: Subscription,
         stream_id: int = 205,
         stream_name: str = "PTEST",
     ) -> None:
         controller.model.narrow = []
         controller.model.index = index_stream
         controller.view.message_view = mocker.patch("urwid.ListBox")
         controller.model.stream_dict = {
-            stream_id: {
+            stream_id: general_stream,
+        }
+        controller.model.stream_dict[stream_id].update(
+            {
                 "color": "#ffffff",
                 "name": stream_name,
             }
-        }
+        )
```

Comment:
It's good that this commit focuses on the stream/subscription migration, but
- it doesn't seem that the color is important to the test(s)?
- that could make this transition smaller/simpler
- can we replace the stream_dict code by setting the return_value in the model, in the later commits? (if we don't already)

Reply:
For the third point, which function's return_value are you referring to?

Reply:
I meant that where you're moving to use the functions that access the name and color, instead of using stream_dict, can we now mock those functions and set the return value directly?

Essentially, we want to ensure we move away from the stream_dict fixture (or equivalent manual dicts), in addition to directly accessing the model attribute.